stdpopsim/catalog/AraTha/genetic_maps.py

```diff
@@ -2,10 +2,18 @@
 
 _species = stdpopsim.get_species("AraTha")
 
+_genetic_map_citation = stdpopsim.Citation(
+    doi="https://doi.org/10.1038/hdy.2011.95",
+    author="Salomé et al.",
+    year=2012,
+    reasons={stdpopsim.CiteReason.GEN_MAP},
+)
+
+tarballname = "AraTha/salome2012_liftover_maps.tar.gz"
 
 _gm = stdpopsim.GeneticMap(
     species=_species,
-    id="SalomeAveraged_TAIR7",
+    id="TAIR10_lifted",  # ID for genetic map, see naming conventions
```

Comment:
Is this consistent with naming conventions?

Reply:
These naming conventions: https://popsim-consortium.github.io/stdpopsim-docs/latest/development.html#naming-conventions

So, why not `SalomeAveraged_TAIR10`?